### ISSUE TITLE
Admin articles accept/reject form

### ISSUE THREAD

**Issue by jottajorge** [phpreboot/website#159]
Depends on #158 New articles on cronjob

Admin will see an articles list, only that ones which are neither accepted nor rejected.
He can see article in iframe, if site support iframe.

If admin decide to add article in magazine, he will see a form for details (Which we currently enter in article import command, submit form and articles will be copied to articles tables and marked as 'accepted' in feed_articles table.

Question: How JSON files generated on live will be committed to git? For now, manually by admins having live access. Will create a new issue for that.

This issue is child of #134 RSS Feed

**Comment by jottajorge** [comment]
Question: Since the articles will be approved and it's details saved in database, do we still need the json files?

**Comment by kapilsharma** [comment]
Answer: Yes, JSON files will be still needed. Reason: Purpose of JSON files is to allow developers to replicate articles for local development. This is obviously important for perfect local setup and replicate live site locally. Thus if we add any article to live, we must add it's JSON file.

However if we can achieve local replication of live site in any other way, JSON files will not be needed.
